docs/local-development.md

```diff
@@ -15,11 +15,19 @@ To follow the manual and to get your local environment running you'll need to ha
 - Node v16.14.x
 - NPM v8.4.x
 
-## Clone the repo
+## Fork our repo
+- Navigate to [bosonprotocol/boson-protocol-contracts on GitHub](https://github.com/bosonprotocol/boson-protocol-contracts)
```

Comment:
I would rather: 
```suggestion
- Navigate to [bosonprotocol/boson-protocol-contracts](https://github.com/bosonprotocol/boson-protocol-contracts) on GitHub.
```
but happy to accept it as written now anyway

Reply:
Fixed.

Reply:
* Issue and PR templates are not added as files in the repo, but rather [configured in the GitHub interface](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository). 
* The [Contribution page](https://docs.bosonprotocol.io/docs/technical_documentation/contracts/code_of_conduct) is currently not in this repo, but appears under Smart Contract docs in the generated documentation site. I pinged @Rhian about moving it into our repo. It mainly talks about behaviors and penalties. It might be ok on its own alongside a more detailed contribution page, which we should flesh out as a team. 
* I think these small changes should go in now, as the repo is open, and we don't want anyone following the current local dev instructions and trying to push branches to this repo for a PR.